apps/studio/components/interfaces/Auth/Hooks/HookCard.tsx

```diff
@@ -0,0 +1,108 @@
+                <span className="text-foreground-light w-20">type</span>
+                <span className="text-foreground">Postgres function</span>
+              </div>
+              <div className="flex flex-row items-center">
+                <span className="text-foreground-light w-20">schema</span>
+                <Input
+                  title={hook.method.schema}
+                  copy
+                  readOnly
+                  disabled
+                  className="input-mono [&>div>div>div>input]:text-xs [&>div>div>div>input]:opacity-100 flex-1"
+                  value={hook.method.schema}
+                  onCopy={() => {}}
+                />
+              </div>
+              <div className="flex flex-row items-center">
+                <span className="text-foreground-light w-20">function</span>
+                <Input
+                  title={hook.method.functionName}
+                  copy
+                  readOnly
+                  disabled
+                  className="input-mono [&>div>div>div>input]:text-xs [&>div>div>div>input]:opacity-100 flex-1"
+                  value={hook.method.functionName}
+                  onCopy={() => {}}
+                />
+              </div>
+            </div>
+          ) : (
+            <div className="flex flex-col w-full space-y-2">
+              <div className="flex flex-row items-center">
+                <span className="text-foreground-light w-20">type</span>
+                <span className="text-foreground">HTTPS endpoint</span>
+              </div>
+              <div className="flex flex-row items-center">
+                <span className="text-foreground-light w-20">endpoint</span>
+                <Input
+                  title={hook.method.url}
+                  copy
+                  readOnly
+                  disabled
+                  className="input-mono [&>div>div>div>input]:text-xs [&>div>div>div>input]:opacity-100 flex-1"
+                  value={hook.method.url}
+                  onCopy={() => {}}
+                />
+              </div>
+              <div className="flex flex-row items-center">
+                <span className="text-foreground-light w-20">secret</span>
+                <Input
+                  copy
+                  title={hook.method.secret}
+                  reveal={true}
+                  readOnly
+                  disabled
+                  className="input-mono [&>div>div>div>input]:text-xs [&>div>div>div>input]:opacity-100 flex-1"
+                  value={hook.method.secret}
+                />
+              </div>
+            </div>
+          )}
+        </div>
+        <div className="flex flex-row gap-2">
+          <Button type="default" disabled={!canUpdateConfig} onClick={() => onSelect()}>
+            Configure hook
+          </Button>
+        </div>
+      </div>
+      <div className="p-4 flex-1">
+        {hook.enabled ? (
+          <Badge className="space-x-1" size="large" variant="brand">
```

Comment:
<img width="172" alt="Screenshot 2024-05-21 at 11 01 21 AM" src="https://github.com/supabase/supabase/assets/8291514/f33ecddd-a717-4551-9e81-5231bde792a1">

Need to add another Badge variant I think.